ansible/roles/kong-api/defaults/main.yml

```diff
@@ -5946,6 +5946,24 @@ kong_apis:
     - name: request-size-limiting
       config.allowed_payload_size: "{{ small_request_size_limit }}"
 
+  - name: contributorSearch
+    request_path: "{{ program_service_prefix }}/v1/contributor/search"
+    upstream_url: "{{ program_service_url }}/program/v1/contributor/search"
+    strip_request_path: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'userTempAdmin'
```

Comment:
Invalid ACL

Reply:
as per the discussion from @pallakartheekreddy and @ahghatol admin will search for multiple users in a single API call and display the necessary details, 

Approving it to have userTempAdmin ACL.